src/com/lftechnology/java/training/dipak/guess/GuessTheNumber.java

```diff
@@ -0,0 +1,84 @@
+package com.lftechnology.java.training.dipak.guess;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/*Guessing a number game application: 
+*application will generate a random no between 1 and 20. 
+*user guess the value by entering from the console
+*Check if the guess value matched with random number or not, if matched show the guessing value, random value and the no. of attempts. 
+ */
+
+public class GuessTheNumber{
+	private final static Logger LOGGER = Logger.getLogger(GuessTheNumber.class.getName());
+	private int theNumber;
+	private int attempts=0;
+	private int lowerBound;
+	private int upperBound;
+	
+	GuessTheNumber(int lowerBound,int upperBound){
+		this.lowerBound = lowerBound;
+		this.upperBound = upperBound;
+	}
+	/**
+	 * <p>This method generates the random number to be guessed.</p>
+	 * @author Dipak Thapa <dipakthapa@lftechnology.com>
+	 */
+	public void generateNumber(){
+		theNumber = (int) (Math.random()*upperBound+lowerBound);
+	}
+	/**
+	 * <p>This method generates the random number to be guessed.</p>
+	 * @param num
+	 * @return true if guessed number matches the random number else false
+	 * @author Dipak Thapa <dipakthapa@lftechnology.com>
+	 */
+	public boolean checkGuess(int num){
+		attempts++;
+		if( num == theNumber ){
+			return true;			
+		}else{
+			return false;
+		}
+	}
+	public static void main(String[] args) {		
+		LOGGER.info("Welcome to Guessing game");
+		boolean giveup = false;
+		int upperBound=20;
+		int lowerBound=1;
+		GuessTheNumber gtn1 = new GuessTheNumber(lowerBound,upperBound);
+		gtn1.generateNumber();
+		Scanner sc = new Scanner(System.in);
+		int guessedNumber;
+		for(;;){
+			label1:for(;;){

```

Comment:
I appreciate  for  using `labeled break`. But, What significance or importance does this brings right now on this block of code. You could have simply used break; to `terminate the for loop` instead of using `labeled break`.

So, `labeled break` are mostly used on multi-leveled  loop. In that case you could use them to break the outer loop as desired. So, Does it sounds or acts like `GOTO` type statement and is it a bad practice? You should look into this yourself.

But, Once again good job I haven't seen it for ages and never used it myself.